modules/wowchemy-plugin-reveal/layouts/slides/baseof.html

```diff
@@ -71,8 +71,12 @@
   {{ end }}
 
   {{/* Initialize slides. */}}
-  {{ $slidejs := resources.Get "js/wowchemy-slides.js" | js.Build (dict "params" (dict "slides" $.Params.slides )) }}
-  <script src="{{ $slidejs.RelPermalink }}"></script>
+  {{ $slidejs := resources.Get "js/wowchemy-slides.js" }} 
+  <script type="module">
+    import { wowchemySlides } from {{ $slidejs.RelPermalink }}
```

Comment:
We shouldn't be writing Go code inside JS code. Not only is it bad practice (especially for modern Hugo versions which offer a lot of JS capabilities), but also it's making the code very difficult for all the other contributors to maintain - when a contributor sees a JS code block, they expect it to be written in JS, not Go.

Reply:
I take the point, but I don't know enough about Hugo to know if this is possible. We have to be able to pass the options  `Params.slides` into the initialization somehow, and the `js.Build` approach that this replaces wasn't doing what it was meant to